src/Types/PubKeyHash.purs

```diff
@@ -115,14 +114,14 @@ instance Show PaymentPubKeyHash where
 
 -- This is needed for `ApplyArgs`. Plutus has an `unPaymentPubKeyHash` field so
 -- don't newtype derive.
-instance DecodeJson PaymentPubKeyHash where
-  decodeJson = caseJsonObject
+instance DecodeAeson PaymentPubKeyHash where
+  decodeAeson = caseAesonObject
     (Left $ TypeMismatch "Expected object")
     ( flip getField "unPaymentPubKeyHash" >=>
-        decodeJson >>> map PaymentPubKeyHash
+        decodeAeson >>> map PaymentPubKeyHash
     )
 
-newtype StakePubKeyHash = StakePubKeyHash PubKeyHash
+newtype StakePubKeyHash = StakePubKeyHash Ed25519KeyHash
```

Comment:
I think this a bad merge, this was recently changed as a newtype over `PubKeyHash` in line with Plutus, `master` still has this change. See https://github.com/Plutonomicon/cardano-transaction-lib/pull/432